lib/plugins/aws/package/lib/mergeIamTemplates.js

```diff
@@ -233,4 +236,8 @@ module.exports = {
       throw new this.serverless.classes.Error('iamManagedPolicies should be an array of arns');
     }
   },
+
+  isLogsDisabled(functionName) {
```

Comment:
Let's remove this method. One time use doesn't seem to justify secluding simple condition.